pkg/registry/rbac/validation/rule_test.go

```diff
@@ -144,7 +187,7 @@ func TestDefaultRuleResolver(t *testing.T) {
 			effectiveRules: nil,
 		},
 		{
-			StaticRoles: staticRoles1,
+			StaticRoles: staticRolesWithSystemTenant,
 			// Same as above but without a namespace. Only cluster rules should apply.
```

Comment:
You mean "same as above but without a tenant", right?

Reply:
this means same as the cases right above it.

Reply:
will add some clarification.

Reply:
this comment is actually a repeat from previous cases. doesn't add much value so it's now removed.